wpilibj/src/athena/java/edu/wpi/first/wpilibj/MultiSpeedController.java

```diff
@@ -0,0 +1,93 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) FIRST 2016-2017. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package edu.wpi.first.wpilibj;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+/**
+ * Allows multiple {@link SpeedController} objects to be linked together.
+ */
+public class MultiSpeedController implements SpeedController {
+
+  private final ArrayList<SpeedController> m_speedControllers = new ArrayList<>();
+
+  /**
+   * Create a new MultiSpeedController with no SpeedControllers.
+   */
+  public MultiSpeedController() {
+
+  }
+
+  /**
+   * Create a new MultiSpeedController with the provided SpeedControllers.
+   *
+   * @param speedControllers The SpeedControllers to add
+   */
+  public MultiSpeedController(SpeedController... speedControllers) {
+    m_speedControllers.addAll(Arrays.asList(speedControllers));
```

Comment:
Just do the for loop, no need to allocate an extra object to make adding an array to a list easier.